mousestyles/ultradian/__init__.py

```diff
@@ -0,0 +1,358 @@
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import statsmodels.api as sm
+import statsmodels.formula.api as smf
+from scipy.stats import chi2
+
+import mousestyles.data as data
+
+
+def aggregate_interval(strain, mouse, feature, bin_width):
+    """
+    Aggregate the interval data based on n-minute time
+    intervals, return a time series.
+
+    Parameters
+    ----------
+    strain: int
+        nonnegative integer indicating the strain number
+    mouse: int
+        nonnegative integer indicating the mouse number
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W", "AS_prob"}
```

Comment:
Did we actually calculate the AS_prob in this function? I think `mousestyles.data.load_intervals` only load